src/components/movie-card/movie-card.tsx

```diff
@@ -44,15 +41,25 @@ export const MovieCard: FC<MovieCardProps> = ({
     setShowEditModal(false);
   };
 
+  const handleDelete = () => {
+    dispatch(deleteMovieAction(movie.id));
+  };
+
   return (
     <div className={styles.card} onClick={onClick}>
-      <img src={img} alt={`${title}-poster`} className={styles.img} />
+      <img
+        src={movie.poster_path}
+        alt={`${movie.title}-poster`}
+        className={styles.img}
+      />
       <div className={styles.descriptionWrappper}>
         <div className={styles.nameWrapper}>
-          <span className={styles.title}>{title}</span>
-          <span className={styles.year}>{year}</span>
+          <span className={styles.title}>{movie.title}</span>
+          <span className={styles.year}>
+            {new Date(movie.release_date).getFullYear()}
```

Comment:
It's better to wrap this expression with `useMemo` to avoid extra `Date` instance re-creation per each render